src/transformers/jasmine-this.test.ts

```diff
@@ -406,6 +406,58 @@ describe('context', () => {
 `
 )
 
+testChanged(
+  'transforms before blocks',
+  `
+  before(function () {
+    this.hello = 'hi';
+  });
+
+  beforeEach(function () {
+    this.goodbye = 'bye';
+  });
+
+  afterEach(function () {
+    console.log(this.hello);
+    console.log(this.goodbye);
+  });
+
+  describe('context', () => {
+      it('should work', function () {
+          console.log(this.hello);
+          console.log(this.goodbye);
+      });
+  });
+`,
+  `
+  let testContext;
+
+  before(() => {
```

Comment:
This is the main focus of this change, on master, this would be a `beforeEach` and the `before` block wouldn't have `testContext` setup.

Reply:
I'm pretty sure there is no `before` hook, this should be `beforeEach` right?

https://jestjs.io/docs/en/api

Reply:
Oh wow, yea. I was thinking jasmine has a `before` block but not sure where I picked up that assumption. I just pushed an update that I think should now be correct?

Reply:
Oh wow, yea. I was thinking jasmine has a `before` block but not sure where I picked up that assumption. I just pushed an update that I think should now be correct?